src/config_magic.h

```diff
@@ -408,7 +405,6 @@ struct MagicConfig {
     long special_types_count;
     struct SpecialConfigStats special_cfgstats[MAGIC_ITEMS_MAX];
     struct InstanceInfo instance_info[INSTANCE_TYPES_MAX]; //count in crtr_conf
```

Comment:
The count from this comment is gone now I believe.

Reply:
game.conf.crtr_conf.instances_count still exists